.github/workflows/diff-change-to-dist.yaml

```
@@ -33,8 +33,16 @@ jobs:
       - name: Generate diff
         id: diff
         run: |
-          git diff -M05 origin/$GITHUB_BASE_REF -- dist \
-            > $GITHUB_WORKSPACE/dist.diff
+          # Using `origin/$GITHUB_BASE_REF` to avoid actually checking out the branch
+          # as all we need is to let Git diff the two references
+          .github/workflows/scripts/dist-diff.sh origin/$GITHUB_BASE_REF $GITHUB_WORKSPACE
+
+      - name: Save diffs as artifacts
```

Probably don't need to mention that we're saving an artifact

Might use a different action in future

```
Verify distribution build
Save test coverage
Run test task
```

```suggestion
      - name: Save distribution diff
```